portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,53 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/*
```

Comment:
This is a JSDoc comment, so it should start with "/**".

Reply:
Fixed